pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -631,8 +631,9 @@ func osdOnSDNFlag(hostnetwork bool, v cephver.CephVersion) []string {
 	// for more details: https://github.com/rook/rook/issues/3140
 	if !hostnetwork {
 		if v.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 2}) {
-			args = append(args, "--ms-learn-addr-from-peer", "false")
+			args = append(args, fmt.Sprint("--ms-learn-addr-from-peer=false"))
```

Comment:
Is this a necessary change?

Reply:
Not for this PR, but the CI has been so slow that I've tried to put more commits in it. Will open a separate PR for this.

Reply:
https://github.com/rook/rook/pull/3389